datafusion/core/src/physical_planner.rs

```diff
@@ -1311,6 +1335,35 @@ impl DefaultPhysicalPlanner {
                         Ok(plan)
                     }
                 }
+                // LogicalPlan::SubqueryAlias(SubqueryAlias())
+                LogicalPlan::RecursiveQuery(RecursiveQuery { name, static_term, recursive_term, is_distinct }) => {
```

Comment:
Looks like it might be due to a regression with backtrace. https://github.com/apache/arrow-datafusion/pull/8620#issuecomment-1877345429

Going to chalk it up to that for now until I have more time to dig in.

Reply:
Or perhaps we need to increase the rust stack size. Running this succeeds 

```shell
RUST_MIN_STACK=104857600 cargo test -p datafusion --test tpcds_planning
```